objmodel/chapter.md

```diff
@@ -1,86 +1,90 @@
-
+<!-- English spelling: "behaviour" line 46 -->
 A Simple Object Model
 ======================
 
 
-Introduction
-----------------
+## Introduction
```

Comment:
I changed all the hyphen-style header markings to hashes because Git's markdown doesn't accept "++++" as a second-level header. I thought it better to use the same system throughout.